src/components/create-filter-form.js

```diff
@@ -107,18 +111,38 @@ const CreateFilterDialog = ({
     };
 
     const handleSave = () => {
-        if (action) {
-            action({
+        const filterType =
+            newListType === ScriptTypes.SCRIPT ? newListType : 'LINE';
+        createFilter(
+            {
                 name: newNameList,
-                type: newListType,
-                isPrivate: filterPrivacy === 'private',
-            });
-            onClose();
-        }
+                type: filterType,
+                transient: true,
+            },
+            newNameList,
+            newListType,
+            filterPrivacy === 'private',
+            activeDirectory
+        ).then((res) => {
+            if (res.ok) {
+                onClose();
+                setCreateFilterErr('');
```

Comment:
reset `setNewListName('')` and setNewListType('SCRIPT') as well maybe ?

Reply:
i created a resetDialog for filter, contingencies and studies that reset all fields with the default values